python/grass/gunittest/testsuite/test_module_assertions.py

```diff
@@ -1,3 +1,4 @@
+%%python
```

Comment:
This is IPython magic. Useful for running content of this file in a Jupyter notebook. Unrelated to running the test.